docs/transfers.rst

```diff
@@ -0,0 +1,296 @@
+
+.. py:currentmodule:: iota
+
+1. Create Transactions
+~~~~~~~~~~~~~~~~~~~~~~
+The first step is to create the individual transaction objects. You have to
+specify ``address`` and ``value`` for each transaction. A negative ``value``
+means spending from ``address``. Furthermore, you can define a ``tag``, and for
+zero-value transactions, a ``message``. ``timestamp`` is usually auto-generated
+by the IOTA libraries.
+
+In PyOTA, use :py:class:`ProposedTransaction` to declare transactions.
+
+2. Create Bundle from Transactions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+A bundle is a collection of transactions, treated as an atomic unit
+when sent to the network. A bundle makes a value (token) transfer possible by
+grouping together input an output transactions.
+
+A bundle always has to be balanced: the sum of ``value`` attributes of the
+transactions in the bundle should always be zero. Transactions in the bundle
+are indexed individually, and also contain information on how many other
+transactions there are in the bundle.
+
+Once complete, a bundle has to be finalized to generate the bundle hash based
+on the `bundle essence`_. The bundle hash is the unique identifier of the
+bundle.
+
+After finalization, input transactions in the bundle need to be signed to prove
+ownership of tokens being transferred.
+
+:py:class:`ProposedBundle` helps you in PyOTA to create bundles, add transactions,
+finalize the bundle and sign the inputs.
+
+3. Select two tips
+~~~~~~~~~~~~~~~~~~
+
+Tips are transactions that are yet to be confirmed by the network. We can
+obtain two tips by requesting them from a node. In PyOTA, :py:meth:`~Iota.get_transactions_to_approve`
+does the job: it returns a ``trunk`` and a ``branch`` :py:class:`TransactionHash`.
+
+Our bundle will validate these two transactions once in the Tangle.
+
+4. Do Proof-of-Work
+~~~~~~~~~~~~~~~~~~~
+
+The bundle has been finalized, inputs have been signed, we have two tips,
+now it's time to prepare the bundle to be attached to the Tangle. All
+transactions reference two other transactions in the Tangle, therefore we need
+to select these references for each transaction in our bundle.
+
+We also know that transactions `within the bundle are linked together`_ through
+their trunk references. So how do we construct the correct bundle structure
+and also reference two tips from the network?
+
+.. figure:: images/bundle-structure.png
+   :scale: 100 %
+   :alt: Bundle structure with four transactions.
+
+   Structure of a bundle with four transactions. Numbers in brackets denote
+   (``currentIndex``, ``lastIndex``) fields. Head of the bundle has index 3,
+   while tail has index 0.
+
+For all non-head transactions in the bundle, the trunk reference is the next
+transaction in the bundle, while the branch reference is the trunk transaction
+hash, one of the tips.
+
+The head transaction is different: the trunk reference is the trunk tip, while
+the branch reference is the branch tip.
```

Comment:
```suggestion
The head transaction is a little bit different.  Its branch reference is the same tip as for the other transactions in the bundle, but since it is the last transaction in the bundle, its trunk reference is the other selected tip.
```

Reply:
Its branch reference is the other tip, its trunk reference is the same tip as for the other transactions.